specification/monitor/resource-manager/Microsoft.Insights/preview/2024-10-01-preview/actionGroups_API.json

```diff
@@ -0,0 +1,1322 @@
+          },
+          {
+            "$ref": "#/parameters/NotificationIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The notification details",
+            "schema": {
+              "$ref": "#/definitions/TestNotificationDetailsResponse"
+            }
+          },
+          "default": {
+            "description": "An error occurred while sending the test notifications",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Insights/actionGroups": {
+      "get": {
+        "description": "Get a list of all action groups in a subscription.",
+        "operationId": "ActionGroups_ListBySubscriptionId",
+        "x-ms-examples": {
+          "List action groups at subscription level": {
+            "$ref": "./examples/listActionGroups.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request succeeded.",
+            "schema": {
+              "$ref": "#/definitions/ActionGroupList"
+            }
+          },
+          "default": {
+            "description": "An error occurred and the list of action groups could not be retrieved.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Insights/actionGroups": {
+      "get": {
+        "description": "Get a list of all action groups in a resource group.",
+        "operationId": "ActionGroups_ListByResourceGroup",
+        "x-ms-examples": {
+          "List action groups at resource group level": {
+            "$ref": "./examples/listActionGroups.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
```

Comment:
>/v3/type [](http://example.com/codeflow?start=68&length=8)

just use the v5 in all these updates as well

Reply:
Fixed. Thank you!